tests/networkpolicy_test.go

```diff
@@ -15,28 +15,23 @@ import (
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
 )
 
-func assertPingSucceed(ip string, vmi *v1.VirtualMachineInstance) {
-	expecter, err := tests.LoggedInCirrosExpecter(vmi)
-	Expect(err).ToNot(HaveOccurred())
-	defer expecter.Close()
-
-	err = tests.CheckForTextExpecter(vmi, []expect.Batcher{
-		&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", ip)},
-		&expect.BExp{R: "0% packet loss"},
-	}, 60)
-	Expect(err).ToNot(HaveOccurred())
+func pingEventually(fromVmi *v1.VirtualMachineInstance, toIp string) AsyncAssertion {
+	return Eventually(func() error {
```

Comment:
Change to `EventuallyWithOffset(2, ....` so after failing ginkgo will point to the failing tests not the helper function.

Reply:
Done